provisioner/k8s/README.md

```diff
@@ -1,3 +1,112 @@
-# k8s
+# k8s Provisioner
 
-A concrete Provisioner implementation for "plain" k8s bundles.
+## Summary
+
+The k8s provisioner is a core rukpak provisioner that knows how to interact with `k8s` format bundles. 
+The `k8s` bundles, or plain bundles, are simply container images containing a set of static kubernetes YAML manifests in a given directory.  
+For example, below is an example Dockerfile that builds a plain `k8s` bundle from a directory containing static manifests.  
```

Comment:
generalized nit: going back to that newline thing, feel free to ignore. It isn't blocking.

```suggestion
The k8s provisioner is a core rukpak provisioner that knows how to interact with `k8s` format bundles. The `k8s` bundles, or plain bundles, are simply container images containing a set of static kubernetes YAML manifests in a given directory.  For example, below is an example Dockerfile that builds a plain `k8s` bundle from a directory containing static manifests.
```